bin/winsvc_wrapper.winsvc.ml

```diff
@@ -0,0 +1,41 @@
+let formatter =
+  let f = Filename.null in
```

Comment:
Is this supposed to be `null`? The logs are pretty useful when things go wrong!

Reply:
I needed to disable stdin, stout and stderr, they're closed for Windows services. We can redirect the logs to a file (what would be a good location?), or maybe use a dedicated function of Windows for logging if it exists?
My googling hasn't returned a meaningful result, maybe @dra27 knows better.